pkgs/tools/audio/jack-tools/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchgit, pkgconfig, libjack2, alsaLib, libGL, libGLU
+,freeglut, libsndfile, libsamplerate, liblo, ncurses, libtool, xorg, asciidoc, libxslt, libxml2 }:
+
+stdenv.mkDerivation rec {
+  name = "jack-tools-${version}";
+  version = "20131226";
+
+  src = fetchgit {
+    url    = https://salsa.debian.org/multimedia-team/jack-tools.git;
```

Comment:
Actually, why not use the [upstream repo](http://rohandrape.net/sw/rju/) with `fetchdarcs`. The debian version seems quite outdated.

Reply:
done